pkgs/applications/misc/keepassx/community.nix

```diff
@@ -110,7 +112,7 @@ stdenv.mkDerivation rec {
     readline
     zlib
   ]
-  ++ lib.optional (stdenv.isDarwin && withKeePassTouchID) darwin.apple_sdk.frameworks.LocalAuthentication
```

Comment:
Are we still using the `darwin` input to this package after this change?

Reply:
@turion , good point. Let me check.

Reply:
@turion , removed the `darwin` input since it was no longer needed.